docs/UserGuide.md

```diff
@@ -134,6 +134,11 @@ Picture 7: `edit 2 schedule meeting on wednesday: edits the 2nd task in Taskell
 
 <!--- @@author A0142130A ---> 
 
+#### Showing calendar view : `calendar` or `cal`
+You can view the calendar for the week on right panel to refer to the dates when adding tasks and scheduling events.<br>
```

Comment:
Add an empty line between #### Showing calendar view : `calendar` or `cal` and You can view BLAH BLAH for consistency